src/main/kotlin/calculator/domain/Calculator.kt

```diff
@@ -0,0 +1,31 @@
+package calculator.domain
+
+/**
+ * 계산을 하는 클래스.
+ * Created by Jaesungchi on 2022.05.21..
+ */
+object Calculator {
+    private val CUSTOM_REGEX = Regex("//(.)\n(.*)")
+    private val DEFAULT_SEPARATORS = listOf(Separator(":"), Separator(","))
+
+    fun getResultOfCalculate(input: String?): Int {
+        // 빈 문장 또는 null이 입력되면 0을 반환 한다.
+        if (input.isNullOrBlank()) return 0
```

Comment:
```suggestion
        if (input.isNullOrBlank()) return 0
```

이미 `isNullOrBlack()`라는 메서드에서 알려주기 때문에, 주석은 없어도 될 것 같습니다